invenio_circulation/utils.py

```diff
@@ -38,6 +38,25 @@ def get_default_loan_duration(loan):
     return 30
 
 
+def get_default_extension_duration(loan):
```

Comment:
If we move the initial values to `config.py` these can be removed.

Reply:
I spoke to the librarians and they're saying it should be possible to calculate these values similarly as for the **checkout**